startup.sh

```diff
@@ -1,4 +1,13 @@
 #!/bin/bash
 
 bundle install
-bundle exec rails s -p 3065
+
+if [[ $1 == "--live" ]] ; then
+  GOVUK_APP_DOMAIN=www.gov.uk \
+  GOVUK_WEBSITE_ROOT=https://www.gov.uk \
+  PLEK_SERVICE_CONTENT_STORE_URI=https://www.gov.uk/api \
+  PLEK_SERVICE_STATIC_URI=assets.publishing.service.gov.uk \
+  bundle exec rails s -p 3065
+else
+  bundle exec rails s -p 3065
+fi
```

Comment:
bundler is only invoked once? it's in the if else block, this also makes it easier to copy out into the command line than the exports (something i am doing frequently currently to change one of the variables)

ideally imo we would be using something like this instead of all this faff
https://github.com/philnash/envyable